lib/generators/administrate/dashboard/dashboard_generator.rb

```diff
@@ -10,7 +10,7 @@ class DashboardGenerator < Rails::Generators::NamedBase
         enum: "Field::String",
         float: "Field::Number",
         integer: "Field::Number",
-        time: "Field::DateTime",
```

Comment:
I was more thinking along the lines of adding a `datetime` here and being able to specify `datetime` in the generators; at least I think that's what this file does if I remember right.